### PR TITLE
Update rancher-monitoring images

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
This PR updates rancher-monitoring v0.0.8 charts to make sure all images are up-to-date.

**Comment by cloudnautique** [comment]
Are there CRD updates?

File: charts/rancher-monitoring/v0.0.8/charts/operator/templates/deployment.yaml [hunk 1]
```diff
@@ -48,7 +48,6 @@ spec:
             - --prometheus-config-reloader={{ template "system_default_registry" . }}{{ .Values.image.prometheusConfigReloader.repository }}:{{ .Values.image.prometheusConfigReloader.tag }}
             - --config-reloader-image={{ template "system_default_registry" . }}{{ .Values.image.configmapReload.repository }}:{{ .Values.image.configmapReload.tag }}
             - --labels={{ .Values.apiGroup }}=true
-            - --crd-apigroup={{ template "operator_api_group" . }}
```
Comment (deniseschannon) [inline charts/rancher-monitoring/v0.0.8/charts/operator/templates/deployment.yaml:51]: Why was this dropped?

**Review by deniseschannon — APPROVED** [review]

File: charts/rancher-monitoring/v0.0.8/charts/operator/templates/deployment.yaml (hunk 1 above)
Comment (StrongMonkey) [reply charts/rancher-monitoring/v0.0.8/charts/operator/templates/deployment.yaml:51]: they dropped the flag in their release. @deniseschannon actually don't merge it, there are some possible chart changes regarding to CRD.

**Comment by deniseschannon** [comment]
Closing as there are potential CRD changes that we need to account for.

**Pull request closed without merging**